build/gulp/tasks/stats.ts

```diff
@@ -121,3 +123,27 @@ task(
     'build:stats:bundle',
   ),
 )
+
+task('stats:save', async () => {
+  const commandLineArgs = _.pick(argv, ['sha', 'branch', 'tag', 'pr', 'build'])
+  const bundleStats = require(paths.docsSrc('bundleStats.json'))[UNRELEASED_VERSION_STRING]
+
+  const options = {
+    method: 'POST',
+    uri: process.env.STATS_URI,
+    body: {
+      sha: process.env.CIRCLE_SHA1,
+      branch: process.env.CIRCLE_BRANCH,
```

Comment:
There values are required as I understand, can we add an assert that they're defined?